src/windows/process.rs

```diff
@@ -768,20 +755,20 @@ unsafe fn get_region_size(handle: &HandleWrapper, ptr: LPVOID) -> Result<usize,
 
 unsafe fn get_process_data(
     handle: &HandleWrapper,
-    ptr: LPVOID,
+    ptr: *const c_void,
     size: usize,
 ) -> Result<Vec<u16>, &'static str> {
     let mut buffer: Vec<u16> = Vec::with_capacity(size / 2 + 1);
     let mut bytes_read = 0;
 
-    if ReadProcessMemory(
+    let ret = ReadProcessMemory(
         **handle,
-        ptr as *mut _,
-        buffer.as_mut_ptr() as *mut _,
+        ptr,
+        buffer.as_mut_ptr().cast(),
         size,
-        &mut bytes_read,
-    ) == FALSE
-    {
+        Some(&mut bytes_read),
+    );
+    if ret.is_err() {
```

Comment:
You don't use `ret` here so might be better to do:

```suggestion
    if ReadProcessMemory(
        **handle,
        ptr,
        buffer.as_mut_ptr().cast(),
        size,
        Some(&mut bytes_read),
    ).is_err() {
```